src/R/Components/Impl/PackageManager/IRPackageManager.cs

```diff
@@ -41,30 +41,6 @@ public interface IRPackageManager : IDisposable {
         Task<IReadOnlyList<RPackage>> GetAvailablePackagesAsync();
```

Comment:
BTW RPackage is the same name as our VS package. I guess at some point we may want to rename VS package to something else.